exchanges/kraken/kraken_wrapper.go

```diff
@@ -358,6 +370,8 @@ func (k *Kraken) FetchTradablePairs(ctx context.Context, a asset.Item) (currency
 		}
 
 		pairs = make([]currency.Pair, 0, len(symbols))
+		limits := make([]order.MinMaxLevel, 0, len(symbols))
```

Comment:
We are doing 2 calls anyway when we update, might as well have this dedicated to the `UpdateOrderExecutionLimits` function because it is common across the other exchange. I know there is going duplication of code but it might be better just from a code maintenance perspective.

Reply:
Okay, so you want to not have UpdateOrderExecutionLimits call FetchTradalePairs?
But still have FetchTradable pairs set limits as well?
That seems redundant to me. There's no negative effect of calling FetchTradablePairs other than having to comment to explain.

On the other hand if you're suggesting duplicating a call to get the symbols then I don't agree at all.
We shouldn't make 2 duplicate API calls literally back to back just to avoid this, right?